packages/lib-classifier/src/store/ResourceStore.spec.js

```diff
@@ -49,7 +48,7 @@ sinon.spy(clientStub.panoptes, 'get')
 describe('Model > ResourceStore', function () {
   before(function () {
     resourceStore = ResourceStore.create(resourcesStub)
-    rootStore = RootStub.create({ resources: resourceStore }, { client: clientStub })
+    RootStub.create({ resources: resourceStore }, { client: clientStub })
```

Comment:
I don't think `RootStub` is used anywhere in this file either?

Reply:
Is it not setting up a mock store for the tests? I didn't check.

Reply:
Test fail with `TypeError: Cannot read property 'panoptes' of undefined` if I remove this so I think it's setting up the mock client.